Weapons.cs

```diff
@@ -6,21 +6,21 @@
 
 namespace TwitchTV {
 	class Weapon {
-		public Weapon( string name, string keyword, int cost, int damage ) {
+		public Weapon( string name, string keyword, string stat, int damage ) {
```

Comment:
stat could be an ENUM { STRENGTH, DEXTERITY, BOTH }, not a string.  Type safety.